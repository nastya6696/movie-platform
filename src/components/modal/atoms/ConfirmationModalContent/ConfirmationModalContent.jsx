import CircleTickIcon from '../../../../assets/svg/circleTickIcon.svg';
import styles from './styles.module.scss';

export const ConfirmationModalContent = () => (
  <>
    <img className={styles.icon} src={CircleTickIcon} alt='circleTickIcon' />
    <h2 className={styles.title}>CONGRATULATIONS!</h2>
    <p className={styles.text}>The movie has been added to database successfully </p>
  </>
)
