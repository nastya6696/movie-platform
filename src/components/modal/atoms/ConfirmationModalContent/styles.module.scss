@import "src/assets/scss/common";

.icon {
  display: block;
  text-align: center;
  width: 66px;
  height: 66px;
  margin: auto;
}

.title {
  @include fontSizes(40px, 48px);
  font-weight: 300;
  color: $white;
  margin-top: 31px;
  text-align: center;
}

.text {
  @include fontSizes(20px, 24px);
  color: $white;
  width: 302px;
  margin: 29px auto 0;
  text-align: center;
}
