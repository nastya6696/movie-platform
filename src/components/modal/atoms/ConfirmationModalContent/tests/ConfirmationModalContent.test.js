import {create} from 'react-test-renderer';
import {ConfirmationModalContent} from "../ConfirmationModalContent";

describe('ConfirmationModalContent', () => {
  it('should create simple ConfirmationModalContent component', () => {
    const component = create(<ConfirmationModalContent />);

    expect(component).toMatchSnapshot();
  })
})
