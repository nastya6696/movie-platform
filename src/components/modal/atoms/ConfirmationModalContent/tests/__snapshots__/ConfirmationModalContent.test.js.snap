// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmationModalContent should create simple ConfirmationModalContent component 1`] = `
Array [
  <img
    alt="circleTickIcon"
    className="icon"
    src="circleTickIcon.svg"
  />,
  <h2
    className="title"
  >
    CONGRATULATIONS!
  </h2>,
  <p
    className="text"
  >
    The movie has been added to database successfully 
  </p>,
]
`;
